File: beecrowd/beecrowd_1216.c
#include <stdio.h>
#include <string.h>
int main()
{
    char str[101];
    int des, c = 0, sum = 0;
    double ave = 0.0;
    while (scanf("%s", &str) != EOF)
    {
        if (scanf("%d", &des) != EOF)
        {
            sum = sum + des;
            c++;
        }
    }
    ave = sum / c;
    printf("%.1lf\n", ave);
    return 0;
}

File: beecrowd/beecrowd_1216.java
import java.util.Scanner;

public class beecrowd_1216 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String str;
        int des, c = 0, sum = 0;
        double ave = 0.0;

        while (scanner.hasNext()) {
            str = scanner.next();
            if (scanner.hasNextInt()) {
                des = scanner.nextInt();
                sum += des;
                c++;
            }
        }

        if (c != 0) {
            ave = (double) sum / c;
        }

        System.out.printf("%.1f\n", ave);
        scanner.close();
    }
}
